Gemfile.lock

```diff
@@ -123,7 +123,7 @@ GIT
       spree_core (>= 1.1)
 
 GIT
-  remote: git://github.com/willrjmarshall/foundation_rails_helper.git
+  remote: https://github.com/willrjmarshall/foundation_rails_helper.git
   revision: 4d5d53fdc4b1fb71e66524d298c5c635de82cfbb
   branch: rails3
   specs:
```

Comment:
@sauloperez reposting this just in case you did not see it. I can replicate the upgrade of `warden` to `1.2.7` when testing on my local machine, and I tend to agree with @andypalmer that hacking the lock file is something we should avoid. A quick look at the ruby 2.3 PR that is mentioned in the release notes suggests to me that it is designed to add compatibility for 2.3, rather than deprecate the use of older versions. Are you ok if we include the `warden` upgrade?

Reply:
Yes, the change is very simple. We'll notice pretty soon if the authentication fails.

Reply:
@sauloperez 👍  thanks

Sorry for the stuff around @andypalmer, could I get you to drop that last commit?

Reply:
@andypalmer are you still working on this? Thanks!

Reply:
@enricostano given the length of time, it's probably better to close this PR and re-raise with the Gemfile change. Otherwise this PR will be 4 commits, of which 3 are noise